jjb/dynamic/scale-ci_watcher.yml

```diff
@@ -0,0 +1,139 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    builders:
+    - shell: |-
+        set -exo pipefail
+
+        # Delete if the repo already exists
+        if [[ -d "$SCALE_CI_WATCHER_REPO_PATH" ]]; then
+          rm -rf  $SCALE_CI_WATCHER_REPO_PATH
+        fi
+
+        # Fetch the code
+        git clone $SCALE_CI_WATCHER_REPO $SCALE_CI_WATCHER_REPO_PATH
```

Comment:
Can this be managed by the scm module?  The module allows parameters I believe.

Reply:
Yes, I will modify it to be managed by the git plugin.